README.md

```diff
@@ -4,49 +4,18 @@
 * [Relay](https://github.com/azure/azure-relay-dotnet)
 
-Azure Service Bus is an asynchronous messaging cloud platform that enables you to send messages between decoupled systems. Microsoft offers this feature as a service, which means that you do not need to host any of your own hardware in order to use it.
-
-Refer to the [online documentation](https://azure.microsoft.com/services/service-bus/) to learn more about Service Bus.
-
-This library is built using .NET Standard 1.3. For more information on what platforms are supported see [.NET Platforms Support](https://docs.microsoft.com/en-us/dotnet/articles/standard/library#net-platforms-support).
-
-## How to provide feedback
-
-See our [Contribution Guidelines](./.github/CONTRIBUTING.md).
-
-## How to get support
-
-See our [Support Guidelines](./.github/SUPPORT.md)
-
-## FAQ
-
-### Where can I find examples that use this library?
-
-[https://github.com/Azure/azure-service-bus/tree/master/samples](https://github.com/Azure/azure-service-bus/tree/master/samples)
-
-### How do I run the unit tests? 
-
-In order to run the unit tests, you will need to do the following:
-
-1. Deploy the Azure Resource Manager template located at [/build/azuredeploy.json](/build/azuredeploy.json) by clicking the following button:
 
-    <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-service-bus-dotnet%2Fmaster%2Fbuild%2Fazuredeploy.json" target="_blank">
-        <img src="http://azuredeploy.net/deploybutton.png"/>
-    </a>
+## We've moved!
 
-    *Running the above template will provision a standard Service Bus namespace along with the required entities to successfully run the unit tests.*
+The Microsoft Azure Service Bus Client for .NET has joined the unified Azure Developer Platform and can now be found in the [Azure SDK for .NET](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/servicebus/Microsoft.Azure.ServiceBus) repository.  To view the latest source, participate in the development process, report issues, or engage with the community, please visit our new home.
 
-1. Add an Environment Variable named `azure-service-bus-dotnet/connectionstring` and set the value as the connection string of the newly created namespace. **Please note that if you are using Visual Studio, you must restart Visual Studio in order to use new Environment Variables.**
+This repository has been archived and is intended to provide historical reference and context for the Microsoft Azure Service Bus Client for .NET 
+  
+[Source code](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/servicebus/Microsoft.Azure.ServiceBus) | [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Azure.ServiceBus/) | [API reference documentation](https://docs.microsoft.com/en-us/dotnet/api/overview/azure/service-bus?view=azure-dotnet) | [Product documentation](https://docs.microsoft.com/en-us/azure/service-bus-messaging/)
```

Comment:
>(https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/servicebus/Microsoft.Azure.ServiceBus) [](start = 13, length = 98)

Also, whereever this points to, there should be a readme file with the details of running the tests / contributing etc

Reply:
There's a full [ReadMe](https://github.com/Azure/azure-sdk-for-net/blob/master/src/SDKs/ServiceBus/data-plane/README.md) that combined content from the initial migration with the standards in place for the central repository.